new/package.json

```diff
@@ -4,9 +4,13 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "antd": "^3.26.3",
     "concurrently": "^5.0.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "css-loader": "^3.4.0",
+    "style-loader": "^1.0.2"
```

Comment:
This is added in package.json here, inside client, and inside server. Remove from the ones inside client and server and make sure the components works perfectly even then.